ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -666,3 +666,193 @@ spec:
   purpose: Informational
   tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_2.3.7.8
   contributors: marcosd4h
+# ---
```

Comment:
code should not be commented, instead you should put TBD on the Query section, and CIS_not_completed on the tags section